core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -148,22 +169,27 @@ export class AdminBackendApiService {
     });
   }
 
-  private _postAdminActionAsync(action:string, payload:Object): Promise<void> {
-    return this.http.post<void>(
-      AdminPageConstants.ADMIN_HANDLER_URL, { action, ...payload }).toPromise()
-      .then(response => {
-        return response;
-      }, errorResonse => {
-        return errorResonse.error.error;
-      });
+  private _postAdminActionAsync(
```

Comment:
Change the name of this function to _postRequestAsync (As this is not only used for action handler.)

Reply:
done!